string id
---
trigger_action_programming/Rule rule
